frontend/src/components/Slider.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class Slider extends Component {
```

Comment:
I'd use stateless functional components:
- cleaner syntax 
- enforces not storing state within component
- will be subject to performance optimisation in the future
https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components